bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Region.java

```diff
@@ -614,8 +621,29 @@ void translateInPixels (int x, int y) {
 public void translate (Point pt) {
 	if (isDisposed()) SWT.error(SWT.ERROR_GRAPHIC_DISPOSED);
 	if (pt == null) SWT.error(SWT.ERROR_NULL_ARGUMENT);
-	pt = DPIUtil.autoScaleUp(pt);
-	translateInPixels(pt.x, pt.y);
+	Point p = DPIUtil.autoScaleUp(pt, initalZoom);
+	translateInPixels(p.x, p.y);
+}
+
+private HashMap<Integer, Long> handleMap = new HashMap<>();
+
+/**
+ * Gets the handle for the region scaled at required zoom level
+ *
+ * @param region the region to be scaled
+ *
```

Comment:
The comment is missing the `@noreference` and the please to not use part

Reply:
Done